### PR TITLE
Updating tests for VSO in EKS

### PR DESCRIPTION
**Pull request by gautambaghel**

### CODE REVIEW

**Comment by gautambaghel** [comment]
Works! And ready to be merged!

File: README.md [hunk 1]
```diff
@@ -134,16 +134,13 @@ make integration-test
...
 # Create an EKS cluster and a ECR repository
-make create-eks
+make -f aws.mk create-eks
 
-# Build the operator binary, image, and deploy to the ECR repository
-make ci-ecr-build-push 
-
-# Run the integration tests (includes Vault OSS deployment)
-make integration-test-eks
+# Build the operator image & run the integration tests (includes Vault OSS deployment)
```
Comment (benashz) [inline README.md:139]:
```suggestion
# Build the operator image and run the integration tests (includes Vault OSS deployment)
```

**Review by benashz — APPROVED** [review]
One minor suggestion to consider, then :+1:

**Review by kschoche — APPROVED** [review]
LGTM!

**Pull request merged**